data/src/main/java/io/gnosis/data/models/transaction/TransactionInfo.kt

```diff
@@ -58,8 +47,16 @@ sealed class TransactionInfo(
 enum class TransactionDirection {
     @Json(name = "INCOMING")
     INCOMING,
+
     @Json(name = "OUTGOING")
     OUTGOING,
+
     @Json(name = "UNKNOWN")
     UNKNOWN
 }
+
+@JsonClass(generateAdapter = true)
```

Comment:
Will we use AddressInfo in other places like settings, advanced settings? If yes it would make sence to move it outside of the transaction-package and place it into data/models